dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/JaxRsIntegration.java

```diff
@@ -125,6 +128,19 @@ public static void setUp() throws Exception {
                         .addAsServiceProvider(ConfigurableSpanExporterProvider.class, InMemorySpanExporterProvider.class)
                         .addAsResource(w3cTraceAppConfig, "META-INF/microprofile-config.properties");
 
+        PropertiesAsset w3cTraceContextAppConfig = new PropertiesAsset()
+                        .include(appConfig)
+                        .addProperty("otel.propagators", "tracecontext");
+        WebArchive w3cTraceContextApp = ShrinkWrap.create(WebArchive.class, W3C_TRACE_CONTEXT_PROPAGATOR_APP_NAME + ".war")
+                        .addClass(W3CTraceContextPropagatorTestServlet.class)
+                        .addPackage(InMemorySpanExporter.class.getPackage())
+                        .addPackage(TestSpans.class.getPackage())
+                        .addPackage(AbstractSpanMatcher.class.getPackage())
+                        //.addPackage(PropagationHeaderEndpoint.class.getPackage())
+                        .addAsServiceProvider(ConfigurableSpanExporterProvider.class, InMemorySpanExporterProvider.class)
+                        .addAsResource(w3cTraceContextAppConfig, "META-INF/microprofile-config.properties");
```

Comment:
```suggestion
        WebArchive w3cTraceContextApp = ShrinkWrap.create(WebArchive.class, W3C_TRACE_CONTEXT_PROPAGATOR_APP_NAME + ".war")
                        .addClass(W3CTraceContextPropagatorTestServlet.class);
```

I think you're putting way more than is necessary into this app.

Given what you have in your test, I don't think OTel even needs to be enabled since we're just calling the W3C propagator manually.